RFS/src/main/java/com/rfs/version_es_6_8/ElasticsearchConstants_ES_6_8.java

```diff
@@ -18,12 +23,13 @@ public class ElasticsearchConstants_ES_6_8 {
 
         // Taken from:
         // https://github.com/elastic/elasticsearch/blob/6.8/libs/x-content/src/main/java/org/elasticsearch/common/xcontent/smile/SmileXContent.java#L55
-        SmileFactory smileFactory = new SmileFactory();
-        smileFactory.configure(SmileGenerator.Feature.ENCODE_BINARY_AS_7BIT, false);
-        smileFactory.configure(SmileFactory.Feature.FAIL_ON_SYMBOL_HASH_OVERFLOW, false);
-        smileFactory.configure(JsonGenerator.Feature.AUTO_CLOSE_JSON_CONTENT, false);
-        smileFactory.configure(JsonParser.Feature.STRICT_DUPLICATE_DETECTION, false);
-        SMILE_FACTORY = smileFactory;
+        SMILE_FACTORY = SmileFactory.builder()
+                .configure(SmileGenerator.Feature.ENCODE_BINARY_AS_7BIT, false)
+                .configure(JsonFactory.Feature.FAIL_ON_SYMBOL_HASH_OVERFLOW, false)
+                .build();
+
+        SMILE_FACTORY.disable(JsonGenerator.Feature.AUTO_CLOSE_JSON_CONTENT);
+        SMILE_FACTORY.disable(JsonParser.Feature.STRICT_DUPLICATE_DETECTION);
```

Comment:
I think that the old way was more clear, to keep all of the lines consistent.  Can you do .configure(..., false)?

Reply:
For whatever reason, using the `.configure()` method with these two `Feature`s was deprecated.